src/ResourceManager/Storage/Commands.Management.Storage.Test/TestController.cs

```diff
@@ -0,0 +1,171 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using Microsoft.Azure.Common.Authentication;
+using Microsoft.Azure.Gallery;
+using Microsoft.Azure.Management.Authorization;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Subscriptions;
+using Microsoft.Azure.Test;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+
+namespace Microsoft.Azure.Commands.Management.Storage.Test.ScenarioTests
+{
+    public class TestController
+    {
+        bool testViaCsm = true; // Currently set to true, we will get this from Environment varialbe.
+
+        private CSMTestEnvironmentFactory csmTestFactory;
+        private EnvironmentSetupHelper helper;
+        
+        public ResourceManagementClient ResourceManagementClient { get; private set; }
+
+        public SubscriptionClient SubscriptionClient { get; private set; }
+        
+        public AuthorizationManagementClient AuthorizationManagementClient { get; private set; }
+
+        public StorageManagementClient StorageClient { get; private set; }
+
+        public GalleryClient GalleryClient { get; private set; }
```

Comment:
I thought this too at the beginning. But the test framework will report an error without GalleryClient initialized. So I can just to put it here to resolve the issue.